_posts/2020-09-17-using-tettra.md

```diff
@@ -0,0 +1,86 @@
+
+## Different Types of Content
+
+When talking about text, we can add content to Tettra directly through a page or a link to an external Google Doc or Sheet. However, when it comes to content in the form of videos, PDF documents, image files or audio, we need a place to store those files.
+
+We use Google Drive for that, so the content is organized and available to the whole team. However, links to those files exist inside a Tettra page so everything is accessible from within the knowledge base. This way, we're able to keep the content centralized.
+
+## Slack Integration
+
+Tettra integrates nicely with [Slack](https://slack.com), our tool of choice for internal communication. Since our team is in Slack a lot, we decided integrating our knowledge base with it was pivotal.
+
+The Slack integration allows us to notify a Slack channel every time a new page is published or an existing page is updated. We have specific categories linked to specific channels and when a page or changes to a page are published, we just need to click a button to notify the team on Slack.
+
+<img src="/blog/assets/images/tettra/slack-notify.jpg" alt="tettra notify slack" class="medium-img">
+
+Through the integration we can also easily search for content. Tettra offers a variety of Slack commands that allow us to search for content, add a new page or create a new suggestion.
+
+Finally, the integration also adds the Tettra Wiki bot, which notifies a user in Slack when they are tagged in a Tettra page or suggestion.
+
+<img src="/blog/assets/images/tettra/slack-notification.jpg" alt="tettra Slack notification" class="medium-img">
+
+## Custom Integration
+
+When we thought of the knowledge base, it was important to us to connect it to our blog posts, as it contains a great deal of information on our procedures and how we use the tools we choose to use.
+
+In order to keep those synced, we created a custom integration via Tettra's API. This way, all the information available in our blog is also accessible from the knowledge base.
+
+## Keeping the Knowledge Base Up to Date
+
+Documentation becomes obsolete, policies change and so on. Adding content is a good way to make sure all the information is safely stored, but removing obsolete content and updating existing pages is also important to keep the knowledge base clean and relevant.
+
+In order to ensure our content is always up to date, we have a set process to review articles in the Knowledge Base every three months. We also get notifications from Tettra when a suggestion hasn't had any activity in a while, which is very helpful.
+
+## Conclusion
+
+In order to keep every team member on the same page and make information accessible to everyone in the easiest way possible, Tettra was our tool of choice. It integrates nicely with the tools we use, has an easy to use interface and allows us to keep the content updated and accessible.
```

Comment:
@ABizzinotto Not sure if it is worth it to mention the other tools that you evaluated? And why we picked Tettra over the others? IIRC it was related to the integrations they had and how they worked with our infrastructure/existing services (Google Drive + GitHub + Slack) ?

Reply:
Hmm... I didn't want to name any other tools but I added a bit more info on why we went with Tettra and a shout out to their support (they were very helpful in the evaluation process).